src/Kmd.Logic.Gateway.Automation/Publish.cs

```diff
@@ -133,18 +133,26 @@ private async Task CreateOrUpdateProducts(Guid subscriptionId, Guid providerId,
         {
             if (entityId.HasValue)
             {
+                var policies = await client.GetAllPoliciesAsync(subscriptionId, entityId.Value, entityType).ConfigureAwait(false);
```

Comment:
Why do you fetch policies? You already have the result from the ValidationResult

Reply:
Removed, not required